### PR TITLE
[fix #269] Add integration test case for cdc

### LINKED ISSUE #269: cdc need more test cases
## Feature Request

**Is your feature request related to a problem? Please describe:**

cdc need more test cases

**Describe the feature you'd like:**

add more test case for cdc

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by zeminzhou**
### What problem does this PR solve?
Add more integration test case for cdc

Issue Number: close #269

Problem Description: **TBD**

### What is changed and how does it work?
Add more integration test case for cdc

### Code changes

- No code

### Check List for Tests

This PR has been tested by at least one of the following methods:
- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)

### Side effects

- No side effects

### Related changes

- No related changes

### CODE REVIEW

**Comment by zeminzhou** [comment]
PTAL~

**Review by pingyu — APPROVED** [review]
LGTM~

File: cdc/tests/integration_tests/autorandom/run.sh [hunk 1]
```diff
@@ -9,14 +9,16 @@ CDC_BINARY=tikv-cdc.test
...
+FALL_BACK=2621440000
 
 function run() {
 	rm -rf $WORK_DIR && mkdir -p $WORK_DIR
 	start_tidb_cluster --workdir $WORK_DIR
 	cd $WORK_DIR
 
 	start_ts=$(tikv-cdc cli tso query --pd=$UP_PD)
-	sleep 10
+	start_ts=$(expr $start_ts - $FALL_BACK)
```
Comment (pingyu) [inline cdc/tests/integration_tests/autorandom/run.sh:21]: The new commits LGTM.

Nit: extract as a utility function.

**Review by haojinming — APPROVED** [review]
LGTM

File: cdc/tests/integration_tests/autorandom/run.sh (hunk 1 above)
Comment (zeminzhou) [reply cdc/tests/integration_tests/autorandom/run.sh]: Yesh, I extracted as a utility function. PTAL~

**Review by pingyu — COMMENTED** [review]
LGTM~

**Pull request merged**